test/integration/analytical_rosenbrock.cpp

```diff
@@ -104,243 +104,45 @@ using LinearSolverTest = micm::LinearSolver<SparseMatrixTest, micm::LuDecomposit
 template<class RatesPolicy>
 using RosenbrockTest = micm::RosenbrockSolver<RatesPolicy, LinearSolverTest>;
 
+template<class RatesPolicy>
+using BackwardEulerTest = micm::BackwardEuler<RatesPolicy, LinearSolverTest>;
+
 TEST(AnalyticalExamples, Oregonator)
 {
-  /*
-   * This problem is described in
-   * Hairer, E., Wanner, G., 1996. Solving Ordinary Differential Equations II: Stiff and Differential-Algebraic Problems, 2nd
-   * edition. ed. Springer, Berlin ; New York. Page 3
-   *
-   * solutions are provided here
-   * https://www.unige.ch/~hairer/testset/testset.html
-   */
-
   auto params = micm::RosenbrockSolverParameters::SixStageDifferentialAlgebraicRosenbrockParameters();
   using OregonatorTest = Oregonator<micm::Matrix<double>, SparseMatrixTest>;
-  auto solver = OregonatorTest::template CreateSolver<RosenbrockTest<OregonatorTest>, LinearSolverTest>(params, 1);
-
-  double end = 360;
-  double time_step = 30;
-  size_t N = static_cast<size_t>(end / time_step);
-
-  std::vector<std::vector<double>> model_concentrations(N + 1, std::vector<double>(3));
-  std::vector<std::vector<double>> analytical_concentrations(13, std::vector<double>(3));
-
-  model_concentrations[0] = { 1, 2, 3 };
-
-  analytical_concentrations = {
-    { 1, 2, 3 },
-    { 0.1000661467180497E+01, 0.1512778937348249E+04, 0.1035854312767229E+05 },
-    { 0.1000874625199626E+01, 0.1144336972384497E+04, 0.8372149966624639E+02 },
-    { 0.1001890368438751E+01, 0.5299926232295553E+03, 0.1662279579042420E+01 },
-    { 0.1004118022612645E+01, 0.2438326079910346E+03, 0.1008822224048647E+01 },
-    { 0.1008995416634061E+01, 0.1121664388662539E+03, 0.1007783229065319E+01 },
-    { 0.1019763472537298E+01, 0.5159761322947535E+02, 0.1016985778956374E+01 },
-    { 0.1043985088527474E+01, 0.2373442027531524E+02, 0.1037691843544522E+01 },
-    { 0.1100849071667922E+01, 0.1091533805469020E+02, 0.1085831969810860E+01 },
-    { 0.1249102130020572E+01, 0.5013945178605446E+01, 0.1208326626237875E+01 },
-    { 0.1779724751937019E+01, 0.2281852385542403E+01, 0.1613754023671725E+01 },
-    { 0.1000889326903503E+01, 0.1125438585746596E+04, 0.1641049483777168E+05 },
-    { 0.1000814870318523E+01, 0.1228178521549889E+04, 0.1320554942846513E+03 },
-  };
-
-  auto state = solver.rates_.GetState();
 
-  state.variables_[0] = model_concentrations[0];
+  auto rosenbrock_solver = OregonatorTest::template CreateSolver<RosenbrockTest<OregonatorTest>, LinearSolverTest>(params, 1);
+  auto backward_euler_params = micm::BackwardEulerSolverParameters();
+  double abolute_tolerance = 1e-16;
+  backward_euler_params.absolute_tolerance_ = { abolute_tolerance, abolute_tolerance, abolute_tolerance };
+  backward_euler_params.relative_tolerance_ = 1e-10;
```

Comment:
Looks like you only override the `backward_euler_params` for this test. Is there a specific reason to do so?

Reply:
I was trying different tolerances to see if I could get backward euler to pass, but nothing I tried seemed to work

Reply:
If we decide to remove the test for OREGO, we probably do not need this change either?

Reply:
agreeed. It's been removed.